CONTRIBUTING.md

```diff
@@ -168,4 +168,23 @@ describe('util/args: ', () => {
 })
 ```
 
+### How to report issues (bugs)
+
+Bug report should contain all the information required to reproduce a bug and to make it as easy as possible to fix. This includes providing the following information
+
+- a description of the error,
+- a description of the expected behaviour,
+- a description of the actual behaviour,
+- steps to reproduce
+
+As an additional info, you can also provide either a Stackblitz link, OR
+
+- the manifest file that generated the error,
+- links to any code (e.g. your own plugin code), it must be available online,
+- runtime information such as OS, node version, package.json, IF version
+
+Reported bugs should pass the triage stage and be prioritized (low, medium or high).Then the issues will be fixed in the next available opportunity post-triage. 
```

Comment:
```suggestion
Reported bugs will be discussed among the team in a weekly bug triage and be assigned a severity (low, medium or high). 

High severity bugs will be fixed as soon as possible, whereas medium and low severity bug fixes will likely be backlogged for attention in the next available sprint. 

In some cases, we might decide not to fix certain bugs if they are low severity, either because we anticipate fixes coming soon as part of already-scheduled upgrades or because we think the fixes make "good first issues" for community contributors.
```